modules/rostests/winetests/msi/msi.c

```diff
@@ -13505,7 +13505,7 @@ static void test_MsiEnumProducts(void)
     ok(r == ERROR_NO_MORE_ITEMS, "Expected ERROR_NO_MORE_ITEMS, got %u\n", r);
     ok(found1, "product1 not found\n");
     ok(found2, "product2 not found\n");
-    ok(found3, "product3 not found\n");
+    flaky_skip ok(found3, "product3 not found\n");
```

Comment:
I'd recommend putting the `flaky_skip` on its own line (as is often done for `todo_wine`) to simplify diffs & syncing.

Reply:
Done